.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -13,9 +13,11 @@ body:
     attributes:
       label: "Steps to reproduce"
       placeholder: |
+        ```
```

Comment:
would adding sh add some basic shell syntax highlighting? might be good to add
```suggestion
        ```sh
```